### DISCUSSION TITLE
I own a company named Ziawe, Inc.

### DISCUSSION THREAD

**Discussion by Caraveo** [community/community#20384] [Other Feature Feedback, Questions, & Ideas]
Someone has taken my ORGANIZATION name.  Ziawe, what can I do about this?  It seems the repository is private.

**Comment by ismohamedi** [comment]
Your Organization Repository is private? or that of the one who took your name?

**Comment by diegoleondev** [comment]
👋Hi @Caraveo 

To report, go to the following form: https://support.github.com/contact/trademark-policy

See the following github article for more information: https://docs.github.com/es/site-policy/content-removal-policies/github-trademark-policy

>**Note**
>Please **Mark as answer** if it was helpful.

**Comment by jfullstackdev** [comment]
> Someone has taken my ORGANIZATION name. Ziawe, what can I do about this? It seems the repository is private.

you mean the repository or the Org ? because as far as I know, the Org itself cannot be hidden but the repos and the members 

https://github.community/t/can-i-creare-a-private-organization/161242/2
